cmd/kpod/ps.go

```diff
@@ -281,20 +284,53 @@ func psOutput(containers []*libkpod.ContainerData, server *libkpod.ContainerServ
 			Labels:     labels,
 			Mounts:     mounts,
 		}
-		output = append(output, params)
+		psOutput = append(psOutput, params)
+	}
+	return
+}
+
+// getJSONOutput returns the container info in its raw form
+func getJSONOutput(containers []*libkpod.ContainerData) (psOutput []psJSONParams) {
+	for _, ctr := range containers {
+		running := false
+		if ctr.State.Status == runningState {
+			running = true
+		}
+
+		params := psJSONParams{
+			ID:               ctr.ID,
+			Image:            ctr.FromImage,
+			Command:          ctr.ImageCreatedBy,
+			CreatedAt:        ctr.State.Created,
+			RunningFor:       time.Since(ctr.State.Created),
+			Status:           ctr.State.Status,
+			Ports:            ctr.Config.ExposedPorts,
+			Size:             ctr.SizeRootFs,
+			Names:            ctr.Name,
+			Labels:           ctr.Labels,
+			Mounts:           ctr.Mounts,
+			ContainerRunning: running,
+		}
+		psOutput = append(psOutput, params)
 	}
+	return
+}
 
-	if len(output) == 0 {
+func psOutput(containers []*libkpod.ContainerData, server *libkpod.ContainerServer, opts psOptions) error {
```

Comment:
Personal twitch.  I really don't like a function that has the same name as variables in the same piece of code.  Could you please consider changing this to createPsOutput, generatePsOutput, generateOutput, or something of your choosing?